nibabel/testing/__init__.py

```diff
@@ -164,3 +166,9 @@ def __init__(self, *args, **kwargs):
         warnings.warn('catch_warn_reset is deprecated and will be removed in '
                       'nibabel v3.0; use nibabel.testing.clear_and_catch_warnings.',
                       FutureWarning)
+
+
+def skipif_no_slow_tests(func):
```

Comment:
Sure.  It feels slightly cleaner to me to do:

```
@runif_extras_has('slow')
def test_one():
    pass

@runif_extras_has('bigmem')
def test_two():
    pass
```

but it's true that it's possible to get the wrong string.  I guess that could fixed with ``if test_type not in ('slow', 'bigmem')`` check in the decorator.  But I don't feel strongly.